benchmark/README.md

```diff
@@ -1,72 +1,77 @@
-    promises-dfilatov-vow.js                      578      125.98
-    callbacks-caolan-async-waterfall.js           634       88.64
-    promises-lvivski-davy.js                      653      109.64
-    promises-calvinmetcalf-lie.js                 732      165.41
-    promises-obvious-kew.js                      1346      261.69
-    promises-ecmascript6-native.js               1348      189.29
-    generators-tj-co.js                          1419      164.03
-    promises-then-promise.js                     1571      294.45
-    promises-medikoo-deferred.js                 2091      262.18
-    observables-Reactive-Extensions-RxJS.js      3201      356.76
-    observables-caolan-highland.js               7429      616.78
-    promises-kriskowal-q.js                      9952      694.23
-    observables-baconjs-bacon.js.js             25805      885.55
+    file                                    time(ms)  memory(MB)
+    promises-bluebird-generator                  229       38.34
+    callbacks-suguru03-neo-async-waterfall       253       47.13
+    callbacks-baseline                           256       27.64
+    promises-bluebird                            292       46.70
+    promises-cujojs-when                         312       65.64
+    callbacks-caolan-async-waterfall             361       47.55
+    promises-ecmascript6-native                  434       97.04
+    promises-then-promise                        436       68.77
+    promises-tildeio-rsvp                        448       90.29
+    generators-tj-co                             468       85.20
+    promises-dfilatov-vow                        665      138.54
+    promises-native-async-await                  671      110.64
+    promises-obvious-kew                         736      105.14
+    streamline-generators                        818       81.04
+    promises-calvinmetcalf-lie                   862      155.03
+    promises-lvivski-davy                        868       87.22
+    observables-pozadi-kefir                    1234      139.05
+    streamline-callbacks                        1366      100.46
+    promises-medikoo-deferred                   1552       98.87
+    observables-Reactive-Extensions-RxJS        2285      233.30
+    promises-kriskowal-q                        8592      289.68
+    observables-caolan-highland                12168      532.08
+    observables-baconjs-bacon.js               33410      681.30
 
     Platform info:
-    Windows_NT 6.1.7601 x64
-    Node.JS 1.1.0
-    V8 4.1.0.14
-    Intel(R) Core(TM) i5-2500K CPU @ 3.30GHz × 4
+    Darwin 17.7.0 x64
+    Node.JS 8.11.1
+    V8 6.2.414.50
+    Intel(R) Core(TM) i5-7360U CPU @ 2.30GHz × 4
 
 bench parallel (`--p 25`)
 
-results for 10000 parallel executions, 1 ms per I/O op
+results for 10000 parallel executions, 1 ms per I/O op `ls ./madeup-parallel/*.js | sed -e 's|\.js||' | xargs node ./performance.js --p 25 --t 1 --n 10000`
```

Comment:
I'm not sure I understand this command - `ls ./madeup-parallel/*.js` ?

Reply:
The assumption is that the command would be run from the `benchmark` directory